trieste/acquisition/rule.py

```diff
@@ -148,6 +148,32 @@ def acquire_single(
             datasets=None if dataset is None else {OBJECTIVE: dataset},
         )
 
+    def update_datasets(
+        self, datasets: Mapping[Tag, Dataset], new_datasets: Mapping[Tag, Dataset]
+    ) -> Mapping[Tag, Dataset]:
+        """
+        Update the datasets with new datasets.
+
+        :param datasets: The current datasets.
+        :param new_datasets: The new datasets.
+        :return: The updated datasets.
+        """
+        # Account for the case where there may be an initial dataset that is not tagged
+        # per region. In this case, only the global dataset will exist in datasets. We
+        # want to copy this initial dataset to all the regions.
+        #
+        # If a tag from tagged_output does not exist in datasets, then add it to
+        # datasets by copying the dataset from datasets with the same tag-prefix.
+        # Otherwise keep the existing dataset from datasets.
+        # TODO: this could mean that when we have a global model, the global dataset
```

Comment:
why TODO?

Reply:
I hadn't tidied that yet, that was part of the "double-check". It is done now. Thanks.